packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/billing-tile/BillingTile.component.tsx

```diff
@@ -27,70 +30,93 @@ type TBillingTileProps = {
 
 export default function BillingTile({ id }: TBillingTileProps) {
   const { t } = useTranslation('dashboard');
-
   const { user, dateTimeFormat } = useCurrentUser();
-  const { data: billingService } = useManagedVcdService(id);
+  const {
+    data: billingService,
+    isLoading,
+    isError,
+    error,
+  } = useManagedVcdService(id);
 
   const nextBillingDate = billingService?.data?.billing?.nextBillingDate;
 
   return (
     <div className="h-fit">
       <DashboardTile
         title={t('managed_vcd_dashboard_service_management')}
-        items={[
-          {
-            id: 'mailingList',
-            label: t('managed_vcd_dashboard_mailing_list'),
-            value: <Description>{user?.email}</Description>,
-          },
-          {
-            id: 'serviceRenew',
-            label: t('managed_vcd_dashboard_service_renew'),
-            value: nextBillingDate ? (
-              <Links
-                onClickReturn={() => {}}
-                label={dateTimeFormat?.format(new Date(nextBillingDate))}
-              />
-            ) : (
-              <span>-</span>
-            ),
-          },
-          {
-            id: 'password',
-            label: t('managed_vcd_dashboard_password'),
-            value: (
-              <div className="flex-wrap">
-                <div className="flex items-center gap-x-2">
-                  <OsdsLink disabled>
-                    {t('managed_vcd_dashboard_password_renew')}
-                  </OsdsLink>
-                  <OsdsTooltip className="flex items-center">
-                    <OsdsIcon
-                      className="cursor-pointer"
-                      name={ODS_ICON_NAME.HELP}
-                      size={ODS_ICON_SIZE.xxs}
-                      color={ODS_THEME_COLOR_INTENT.text}
+        items={
+          isError
+            ? [
+                {
+                  id: 'error',
+                  value: (
+                    <OsdsMessage type={ODS_MESSAGE_TYPE.error}>
+                      {error?.response?.data?.message}
+                    </OsdsMessage>
+                  ),
+                },
+              ]
+            : [
+                {
+                  id: 'mailingList',
+                  label: t('managed_vcd_dashboard_mailing_list'),
+                  value: isLoading ? (
+                    <OsdsSkeleton />
+                  ) : (
+                    <Description>{user?.email}</Description>
+                  ),
```

Comment:
Why should we display Skeleton when isLoadingBillingServices while we might have user's data ?

Reply:
I improved this part to only impact service renew bloc